services/slam/fake/slam_test.go

```diff
@@ -32,8 +32,8 @@ func TestFakeSLAMGetPosition(t *testing.T) {
 	// in floating point values between M1 mac & arm64 linux which
 	// were causing tests to pass on M1 mac but fail on ci.
 	expectedPose := spatialmath.NewPose(
-		r3.Vector{X: -0.005666600181385561, Y: -6.933830159344678e-10, Z: -0.013030459250151614},
-		&spatialmath.Quaternion{Real: 0.9999999087728241, Imag: 0, Jmag: 0.0005374749356603168, Kmag: 0})
+		r3.Vector{X: -0.005839621552532072747133, Y: 0.012640521020122929413132, Z: 0.000000000000000000},
+		&spatialmath.Quaternion{Real: 0.9999998930633826, Imag: 0, Jmag: 0, Kmag: -0.000462464294427742})
```

Comment:
Is this definitely the actual orientation you want? It is sooooo close to (1,0,0,0) in which case you could just use NewPoseFromPoint and leave off the quaternion entirely. My guess is that PoseAlmostEqual will still be true

Reply:
I'd prefer to keep the explicit definitions because the test is loading the first pointcloud and in future datasets this first pointcloud may include some small degree of rotation and this will be easier to update with new datasets if we leave these written out in longhand.

@nicksanford @EshaMaharishi  I'd like to hear your thoughts on this.
Peter is correct we can use the default orientation here. Although, we can't use the full: `spatialmath.NewZeroPose()` as the `PoseAlmostEqual` function still fails when comparing the r3.vector from the first fake position dataset defined above with (0,0,0).

Reply:
`NewPoseFromPoint` is a convenience function that will accept your r3.Vector and fill in the zero orientation automatically

Reply:
To be clear; `NewPoseFromPoint` doesn't ignore orientation, it simply auto-fills (1,0,0,0), which is functionally the same as what's currently used. So this would still break if the tests dataset does.

Totally optional change though.